.github/workflows/main.yml

```
@@ -0,0 +1,40 @@
+name: TerminalImageViewer
+
+on:
+  push:
+    branches:
+    - '*'
+    paths-ignore:
+      - '**/README.md'
+  pull_request:
+    branches:
+    - '*'
+  workflow_dispatch:
+
+jobs:
+  image:
+    name: build ${{ matrix.arch_name }} ${{ matrix.build_type }}
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ${{ matrix.arch_name }}
```
This is what I assume to be the corret syntax for building on multiple OSes. Correct me if I'm wrong.